Mathlib/Data/Vector/Basic.lean

```diff
@@ -511,9 +511,22 @@ def casesOn₃ {motive : ∀ {n}, Vector α n → Vector β n → Vector γ n 
     motive v₁ v₂ v₃ :=
   inductionOn₃ (C := motive) v₁ v₂ v₃ nil @fun _ x y z xs ys zs _ => cons x y z xs ys zs
 
-/-- Cast a vector to an array. -/
-def toArray : Vector α n → Array α
```

Comment:
I think this can be `def toArray v := v.1.toArray`

Reply:
Agreed, but I removed it on Eric's suggestion anyway.

Reply:
What's the reason to remove it? I don't see the suggestion. I think it needs a deprecation if so

Reply:
Oh, I thought this was being added anew in this PR, not just relocated.

Reply:
Weird, I could have sworn it was there. I'll add it back! It had a less-good definition before and I thought it seemed best to explicitly link to these new functions.